### ISSUE TITLE
Add --plugin option for CLI

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#1112]
The `--plugin` option should work the same was as `--env` in that it can be used any number of times to specify any number of plugins to load. Such as:

```
eslint --plugin foo --plugin bar test.js
```

These values should be passed as an array to the `CLIEngine` constructor as `plugins`.

**Comment by t128** [comment]
hi, is there a way to disable a plugin?

Im using *eslint-config-airbnb* but i don't need the react plugin. So i would disable it, but don't know how i should do it.

```yaml
extends: eslint-config-airbnb
plugins: []
```
